itenium-socks/src/app/time-ago.pipe.ts

```diff
@@ -0,0 +1,32 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'timeAgo',
+  standalone: true
+})
+export class TimeAgoPipe implements PipeTransform {
+
+  transform(value: any): string {
```

Comment:
`any` ???

We kunnen TimeAgo doen op een array?
Dan ga je de code wel moeten uitbreiden!!

```
transform(value: number | Date | string): string {}
```